_episodes/07-testing.md

```diff
@@ -311,7 +311,7 @@ def test_build_bond_list():
 
     assert len(bonds) == 4
 
-    for atoms, bonds in bonds.items():
+    for bonds in bonds.values():
```

Comment:
I expect this works, but it seems like bad coding style. If you attach a debugger at the assertion, you won't be able to inspect the original `bonds` object. Suggest: `for bond in bonds.values()`

Reply:
Good call, @eirrgang !! It has been a long time since I looked at this lesson. I assume we can change it to:
```python
for bond_lengths in bonds.values():
```
perhaps?

Reply:
singular, though, right?
```suggestion
    for bond_length in bonds.values():
```

Reply:
I concur!! Thank you for checking it!